lib/services/livesync/livesync-command-helper.ts

```diff
@@ -15,13 +15,17 @@ export class LiveSyncCommandHelper implements ILiveSyncCommandHelper {
 		return availablePlatforms;
 	}
 
+	/**
+	 * Method sets up configuration, before calling livesync and expects that devices are already discovered.
+	 * @param devices List of discovered devices
+	 * @param liveSyncService Service expected to do the actual livesyncing
+	 * @param platform The platform for which the livesync will be ran
+	 */
```

Comment:
The JSDocs are more applicable for the interface (`ILiveSyncCommandHelper`) as we are calling the method through it. In order to have autocomplete and JSDocs when using the injected dependency ("liveSyncCommandHelper"), the JSDocs must be in the interface itself.

Also you have to define the return type of the method:
```
* @returns {Promise<void>}
```